trip_app/accounts/tests/test_permissions.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from accounts import views
+from django.contrib.auth.models import AnonymousUser
+from django.urls import reverse
+
+from .user_factory import UserFactory
+
+
+@pytest.mark.django_db
+def test_user_detail_without_perm(rf):
+    """ Verify that client is unable to access GET /api/accounts/<id> if he is not admin or owner """
+    user = UserFactory()
+    url = reverse("accounts:user", kwargs={'pk': user.id})
```

Comment:
Not sure if using `reverse` in test cases is a good practice. I'd expect tests to be as straightforward as possible. Please provide reasons why you choose this approach.

Reply:
If url pattern changes I will have to fix tests in multiple places, this is the main reason why I consider 'reverse' appropriate here. Also official django docs using it as well...

Reply:
1. reasons are now clear
1. still not agreed that those considerations are worth accounting for
1. you can keep it the way you prefer